src/content/docs/de/contribute.mdx

```diff
@@ -38,16 +40,11 @@ Wir haben auch eine sehr aktive [Discord](https://astro.build/chat)-Gemeinschaft
 
 ## Zu den Docs beitragen
 
-Wir haben mehrere Anleitungen mit Informationen darüber, wie du zu Astro Docs beitragen kannst!
+Wir haben mehrere Leitfäden, die dir helfen, zu Astro Docs beizutragen.
 
-- [Main Contributing Guide](https://github.com/withastro/docs/blob/main/CONTRIBUTING.md) - Bitte lies diesen Leitfaden zuerst!
-- [Writing Guide](https://contribute.docs.astro.build/guides/writing-style/) - Tipps zum Schreiben im Astro Docs-Stil.
-- [Translating Guide](https://contribute.docs.astro.build/guides/i18n/) - Informationen darüber, wie du Astro Docs in deine Sprache übersetzen kannst.
-- [Recipe Guide](https://contribute.docs.astro.build/guides/recipe-writing/) - Lerne, wie man eine Astro-Anleitung schreibt und beisteuert.
+Egal, ob es sich um deinen allerersten Beitrag zu Open-Source handelt, du Dokumente für das neue Astro-Feature, das du gerade gebaut hast, hinzufügen musst, ein erfahrener Übersetzer auf der Suche nach der nächsten zu übersetzenden Seite bist oder mehr darüber erfahren möchtest, wie du als PR-Reviewer helfen kannst... wir haben alles für dich! 
 
-Dies sind Anleitungen für das Docs-Repository für bestimmte Rollen innerhalb der Astro-Community.
-- [Maintainer Guide](https://contribute.docs.astro.build/roles/maintainers/) - Wie man dieses Repository als Astro-Maintainer pflegt.
-- [Support Squad Guide](https://contribute.docs.astro.build/roles/support-squad/) - Wie du dem Team Docs als Teil deiner Support-Rolle helfen kannst.
+Bitte besuche unsere spezielle Seite [*Astro Docs* Docs] (https://contribute.docs.astro.build). Dort findest du unsere Dokumentation, die dir dabei hilft, als Tippfehler-Beseitiger, Autor, Übersetzer, Feature-Builder und sogar als PR-Reviewer zu Astro Docs beizutragen.
```

Comment:
```suggestion
Bitte besuche unsere spezielle Seite [*Astro Docs* Docs](https://contribute.docs.astro.build). Dort findest du unsere Dokumentation, die dir dabei hilft, als Tippfehler-Beseitiger, Autor, Übersetzer, Feature-Builder und sogar als PR-Reviewer zu Astro Docs beizutragen.
```

No link was added to the text due to space :)

Reply:
oopsy 🌵